doc/manual/manual.tex

```diff
@@ -6310,11 +6310,78 @@ \subsubsection{The van Keken thermochemical composition benchmark}
+	\end{tabular}
+	\caption{\it Average runtimes for the Van Keken Benchmark with smoothed initial conditions.  These times are for the entire computation, a final time step of 2000.  All of these tests were run using ASPECT version 1.3 in release mode, and used different numbers of processors on one node on the BlueRidge cluster of ARC at Virginia Tech.}
+	\label{tab:runtime-table}
+\end{table}
+
+This convergence is due to changing the smoothing parameter, which controls how much of the problem is smoothed over.  As the parameter is increased. the smoothed boundary grows and vice versa.  As the smoothed boundary shrinks it becomes sharper until the original discontinuous behavior is revealed.  As it grows, the two layers eventually become one large, transitioning layer rather than two distinct layers separated by a boundary.  These effects can be seen in Fig.~\ref{fig:vk-5}.  The overall effect is that the secondary rise is at different times based on these conditions.  In general, as the smoothing parameter is decreased, the smoothed boundary shrinks and the plumes rise more quickly.  As it is increased, the boundary grows and the plumes rise more slowly.  This trend can be used to force a more accurate convergence from the secondary plumes.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.8\textwidth]{cookbooks/benchmarks/van-keken/smoothing-parameter.png}
+	\caption{\it Van Keken Benchmark using smoothed out interface: compositional field at time t=0 using smoothing parameter size: a) 0.0039, b) 0.0078, c) 0.0156, d) 0.0234, e) 0.0312, f) 0.0390, g) 0.0468, h) 0.0546, i) 0.0624.}
+	\label{fig:vk-5}
+\end{figure}
+
+The evolution in time of the resulting compositional fields (Fig.~\ref{fig:vk-6}) shows that the first peak converges as the smoothed interface decreases. There is a good agreement for the first peak for all smoothing parameters.  As the width of the discontinuity increases, the second peak rises both later and more slowly.
+
+\begin{figure}
+	\centering
+	\includegraphics[width=0.75\textwidth]{cookbooks/benchmarks/van-keken/smoothing-parameter-velocity.png}
+	\caption{\it Van Keken benchmark with smoothed initial conditions for the compositional field using refinement degree 7 for different smoothing parameters.}
+	\label{fig:vk-6}
+\end{figure}
+
+Now let us further add a two-layer viscosity model to the domain. Let's assume the viscosity value of the upper heavier layer is $\eta_{t}$ and the viscosity value of the lower lighter layer is $\eta_{b}$. Based on the initial constant viscosity value 1$\times10^{2}$ Pa~s, we set the viscosity proportion $\frac{\eta_{t}}{\eta_{b}}=0.1, 0.01$, meaning the viscosity of the upper, heavier layer is still 1$\times10^{2}$ Pa~s, but the viscosity of the lower, lighter layer is now either 10 or 1 Pa~s, respectively. The viscosity profiles of the discontinuous and smooth models are shown in Fig.~\ref{fig:vk-7}.
```

Comment:
"Now let us further..." -- I assume this corresponds to one of the other benchmarks in the van Keken et al. paper? If so, can you add a half-sentence that this is so (and thereby explain *why* you want to make this modification)?